public/apps/account/test/__snapshots__/log-out-button.test.tsx.snap

```diff
@@ -5,7 +5,7 @@ exports[`Account menu - Log out button renders renders when auth type is OpenId
   <EuiButtonEmpty
     color="danger"
     data-test-subj="log-out-1"
-    href="/app/kibana/auth/logout"
+    href="/auth/logout"
```

Comment:
This does not look like pure organization change. Is this intentional?

Reply:
Yes. This is updated snapshot. I have also updated description.